src/caustics/lenses/tnfw.py

```diff
@@ -394,29 +522,50 @@ def mass_enclosed_2d(
         **kwargs,
     ) -> Tensor:
         """
-        Total projected mass (Msol) within a radius r (arcsec).
+        Total projected mass (solMass) within a radius r (arcsec).
 
         Parameters
         -----------
         z_l: Tensor
             Redshift of the lens.
+
+            *Unit: unitless*
+
         x0: Tensor
-            Center of lens position on x-axis (arcsec).
+            Center of lens position on x-axis.
+
+            *Unit: arcsec*
+
         y0: Tensor
-            Center of lens position on y-axis (arcsec).
+            Center of lens position on y-axis.
+
+            *Unit: arcsec*
+
         mass: Optional[Tensor]
-            Mass of the lens (Msol).
+            Mass of the lens.
+
+            *Unit: solMass*
+
         scale_radius: Optional[Tensor]
-            Scale radius of the TNFW lens (arcsec).
+            Scale radius of the TNFW lens.
+
+            *Unit: arcsec*
+
         tau: Optional[Tensor]
-            Truncation scale. Ratio of truncation radius to scale radius (rt/rs).
+            Truncation scale. Ratio of truncation radius to scale radius.
+
+            *Unit: unitless*
+
         params: dict
             Dynamic parameter container.
 
         Returns
         -------
         Tensor
             Integrated mass projected in infinite cylinder within radius r.
+
+            *Unit: arcsec*
```

Comment:
Msol

Reply:
This has been changed to `Msun` not the suggested Msol per discussions.